.github/settings.yml

```diff
@@ -0,0 +1,2 @@
+repository:
+  name: strappy-doo
```

Comment:
I had that originally, but thought `scrappydoo` is way too long of a name for the CLI, so thought the dash made it easy to just use `scrappy` as the CLI name, and `-Doo` just helps make it googleable. Also, it's pays [homage](https://en.wikipedia.org/wiki/Scrappy-Doo).

What do you think? If it's not weird to call it `scrappydoo` but have the CLI be `scrappy`, I could be convinced to change it.

Reply:
I understand where you're coming from - that's certainly a valid concern. However, I think that `strappy` as a CLI input is still viable; I believe that `doo` is seen as a suffix, therefore it will be often omitted when typed or thought of quickly. But that's just one `doo`er's opinion.